public/style.css

```diff
@@ -0,0 +1,21 @@
+html{
```

Comment:
For all of your styles, you'll want to include a space between your selector and your first brace:
`html {`